stable/phpmyadmin/templates/deployment.yaml

```diff
@@ -75,12 +75,18 @@ spec:
             - name: DATABASE_ENABLE_SSL
               value: {{ ternary "yes" "no" .Values.db.enableSsl | quote }}
             {{- if .Values.db.enableSsl }}
+            {{- if not (empty .Values.db.ssl.clientKey) }}
```

Comment:
it's weird the config parameter are named `clientKey` and `clientCertificate` and mounted files is named `server_key.pem` and `server_certificate.pem` resp. Maybe I am missing something.

Reply:
Well I wouldn't change that to avoid introducing major changes